core/message_queue.go

```diff
@@ -121,6 +118,20 @@ func (mq *MessageQueue) LargestNonce(sender address.Address) (largest uint64, fo
 	return 0, false
 }
 
+// Queues returns the addresses associated with each non-empty queue.
```

Comment:
maybe note that there is no guarantee the addresses are in any particular or consistent order (bc the map iteration).